connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -114,31 +114,20 @@ public Uri getAuthorizeUri(Map<String, String> parameters, List<String> locales)
     @Override
     public DoNext onAuthorize(Map<String, String> parameters) {
```

Comment:
I think that this should be really onPreAuthorize , onPrepareAuthorize or something like that.
onAuthorize implies that the event happens after the authorization.